src/bin/server.rs

```diff
@@ -37,6 +38,7 @@ fn main() -> anyhow::Result<()> {
         .databases_from_config(&config.db)
         .github(github)
         .github_oauth_from_config(&config)
+        .trustpub_providers(&list("TRUSTPUB_PROVIDERS")?)
```

Comment:
this essentially acts as a feature flag for each of the providers. it allows us to enable e.g. `github` on the staging environment, but not yet on production. once we add new providers it will also allow us to gradually enable them.

if a provider is breached it would also allow us to temporarily disable them.